app/controllers/letters_controller.rb

```diff
@@ -35,6 +35,20 @@ def update
     end
   end
 
+  def search
+    to = params[:to].to_date || Date.today
+    from = params[:from].to_date || to - 1.month
+    stat = if params[:stat] == ''
```

Comment:
@letters = current_user.letters.where(created_at: from..to)
@letters = @letters.where(letter_status: stat) if  params[:stat].present?